scheduler/tasks/submission-releaser-task/src/main/kotlin/uk/ac/ebi/scheduler/releaser/service/SubmissionReleaserService.kt

```diff
@@ -41,8 +41,13 @@ class SubmissionReleaserService(
             .forEach(::generateFtpLink)
     }
 
+    private fun releaseSafely(releaseData: ReleaseData) {
+        runCatching { releaseSubmission(releaseData) }
+            .onFailure { logger.info { "Failed to release submission ${releaseData.accNo}" } }
```

Comment:
This is a great improvement, we definitively shouldn't block all the other submissions.

I wonder if we should add this to another queue or maybe send a slack notification. Otherwise, we won't actually realize there are failing submissions for release until the users complain. WDYT?

Reply:
To be address in https://www.pivotaltracker.com/story/show/183688670